modules/samples/pom.xml

```diff
@@ -63,6 +63,7 @@
         <module>TomcatWebApps/CarbonDataSourceAccessExample</module>
         <module>EntitlementFilter</module>
         <module>SSOValveSample</module>
+        <module>CarbonSaaSTest</module>

```

Comment:
Need to add the sample to modules/distribution/src/assembly/bin.xml to be shipped with the binary pack.